low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -75,6 +91,14 @@ def get_health_info(self, rpath):
         fru_found = False
         nodes = rpath.strip().split(">")
         leaf_node, _ = self.get_node_details(nodes[-1])
+        try:
```

Comment:
This should be again in a function, under OS & Build class

Reply:
Moved function to `BuildInfo` class under low-level/framework/platforms/server/software.py